### DISCUSSION TITLE
遥感图文问答大模型

### DISCUSSION THREAD

**Discussion by biscuit279** [InternLM/Tutorial#482] [project]
用xtuner在遥感数据集上进行微调，实现遥感图像的问答
主要想做到遥感图像的开集场景分类

**Comment by YanxingLiu** [comment]
> 用xtuner在遥感数据集上进行微调，实现遥感图像的问答 主要想做到遥感图像的开集场景分类

xd能问下你们满人了吗？

**Reply by biscuit279** [reply]
我就想自己玩玩 你要来一起吗 可以加我一下 微信号biscuit279
